trackers.json

```diff
@@ -57,7 +58,8 @@
     {"Name": "PuroVicio", "Abbreviation": "PRV", "Type": "Movie/TV", "Codebase": "UNIT3D", "Users": "2,720", "Torrents": "3,073", "Peers": "3,598", "Ratio": "Yes", "Ratio Diff": "-", "Points": "Yes", "Birthdate": "2018", "Join": "Invite/Open", "Join Diff": "Easy", "Updated": "12/29/19"},
     {"Name": "PsyTorrents", "Abbreviation": "PsyT", "Type": "Music", "Codebase": "Gazelle", "Users": "1,757", "Torrents": "13,122", "Peers": "49,316", "Ratio": "Yes", "Ratio Diff": "-", "Points": "No", "Birthdate": "-", "Join": "Invite", "Join Diff": "-", "Updated": "01/01/20"},
     {"Name": "Racing4Everyone", "Abbreviation": "R4E", "Type": "Sports", "Codebase": "UNIT3D", "Users": "3,402", "Torrents": "4,423", "Peers": "9,690", "Ratio": "Yes", "Ratio Diff": "-", "Points": "Yes", "Birthdate": "-", "Join": "Invite/Open", "Join Diff": "-", "Updated": "12/29/19"},
-    {"Name": "Redacted", "Abbreviation": "RED", "Type": "Music", "Codebase": "Gazelle", "Users": "34,813", "Torrents": "1,778,901", "Peers": "5,042,899", "Ratio": "Yes", "Ratio Diff": "-", "Points": "No", "Birthdate": "2016", "Join": "Application/Invite", "Join Diff": "-", "Updated": "12/29/19"},
+    {"Name": "Redacted", "Abbreviation": "RED", "Type": "Music", "Codebase": "Gazelle", "Users": "34,813", "Torrents": "1,778,901", "Peers": "5,042,899", "Ratio": "Yes", "Ratio Diff": "-", "Points": "No", "Birthdate": "-", "Join": "Application/Invite", "Join Diff": "-", "Updated": "12/29/19"},
```

Comment:
@HDVinnie It seems that when the merger was accidentally deleted, the foundation year of RED.

Reply:
fixed in c617397fee55c76600a8fefb42ec369aaad27343

Reply:
@HDVinnie thanks!